### DISCUSSION TITLE
Move fields from one collection to another

### DISCUSSION THREAD

**Discussion by twiddlecoding** [pocketbase/pocketbase#4378] [Q&A]
Hello!
I need to move some fields from one collection to another collection. There needs to be a relational field that binds the records from these two collections to each other. How would you go about doing such a batch job? The database is in a production enviroment, hosted on PikaPods.

**Comment by ganigeorgiev** [comment]
The recommended approach to apply schema and data changes across different environments is to write migrations:

- Go: https://pocketbase.io/docs/go-migrations/

- JS `pb_hooks`: https://pocketbase.io/docs/js-migrations/

If you need help with the migration, please provide more details about your existing and desired collections structure.
